src/pages/EntityDetails/Model/Logs/ActionLogs/_action-logs.scss

```diff
@@ -1,21 +1,33 @@
 @import "../../../../../scss/settings";
 @import "vanilla-framework/scss/vanilla";
+@import "../../../../../scss/breakpoints";
 
 @mixin action-logs {
   .entity-details__action-logs {
-    table {
-      // Operation ID column
-      th:nth-child(2) {
+    td,
+    th {
+      // Completion time column.
```

Comment:
Small nitpicky remark: Was wondering if maybe we can standardize the way we write these comments and specify only the name of the column here instead, as was done in #1657 and #1658?